### ISSUE TITLE
phpcompiler.org leads to a compromised site

### ISSUE THREAD

**Issue by perk32725** [taichino/php2py#6]
Looks like phc might be dead...

**Comment by EpicCodeWizard** [comment]
Yes. Use this instead: https://github.com/pbiggar/phc. Apparently, they forget to renew their ownership of the domain. I still can't figure out how to install it, I am on windows.
